vnext/ReactUWP/Views/Image/ImageViewManager.cpp

```diff
@@ -129,15 +107,24 @@ namespace react { namespace uwp {
 
   XamlView ImageViewManager::CreateViewCore(int64_t tag)
   {
-    winrt::Image image;
-    image.Stretch(winrt::Stretch::UniformToFill);
-    return image;
+    auto reactImage{ ReactImage::Create() };
+
+    reactImage->OnLoadEnd([=](const auto&, const bool& succeeded)
+    {
+        ImageSource source{ reactImage->Source() };
+
+        EmitImageEvent(m_wkReactInstance.lock(), reactImage.as<winrt::Canvas>(), succeeded ? "topLoad" : "topError", source);
+        EmitImageEvent(m_wkReactInstance.lock(), reactImage.as<winrt::Canvas>(), "topLoadEnd", source);
```

Comment:
as mentioned over chat, we want to change these to fire only when subscribed to, which will require having a shadownode to store if onLoad/etc have been set.

Reply:
oh i thought maybe these were firing more than before your change (for non-http) but that doesn't appear to be the case, future fix...

---
In reply to: [294093989](https://github.com/microsoft/react-native-windows/pull/2581#discussion_r294093989) [](ancestors = 294093989)